cellxgene_resources/curation_sample_code.ipynb

```diff
@@ -748,29 +748,21 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Extracts gene_id and gene_symbol from gtf\n",
+    "#Read in Gencode-formatted gtf file\n",
+    "genome_version = ''\n",
+    "ref_file = f's3://submissions-lattice/reference-files/refdata-gex-{genome_version}.gtf'\n",
```

Comment:
The GENCODE-formatted gtf files won't have consistent naming conventions (only 2 of the CellRanger outputs would fit the proposed pattern). And they will likely be local, not read directly from S3.
So would suggest:

- removing `genome_version` line
- moving `my_dir = 'path/to/working/dir'` up here (`dir` is a built-in function so I'd avoid using that variable)
- then `ref_file = my_ref.gtf` (as a placeholder, like the dir)